misk-hotwire/src/main/resources/web/static/cache/README.md

```diff
@@ -0,0 +1,7 @@
+# /web/static/cache
+
+The few web dependencies for the admin dashboard and other frontend apps don't change frequently. 
+
+To support easier local development, dependencies are checked in to the repo so the admin dashboard is available without `npm install` or any non-Gradle steps.
```

Comment:
already posted but why not use a cdn?

Reply:
referring back to my prior response: https://github.com/cashapp/misk/pull/2800#discussion_r1210170569